Forge.TreeWalker.UnitTests/test/TreeWalkerUnitTests.cs

```diff
@@ -153,6 +184,19 @@ public void TestTreeWalkerSession_VisitNode_Success()
             Assert.AreEqual(expected, actualNextTreeNodeKey, "Expected VisitNode(Container) to return Tardigrade.");
         }
 
+        [TestMethod]
+        public void TestTreeWalkerSession_VisitNode_Success_WithV1Callbacks()
```

Comment:
>TestTreeWalkerSession_VisitNode_Success_WithV1Callbacks [](start = 20, length = 55)

This test calls VisitNode directly, which avoids use of the ITreeWalkerCallbacks completely. Instead, copy the TestTreeWalkerSession_WalkTree_Success test.

Reply:
fixed. and updated a few test methods to cover V1 with File/ForgeTree, and V2 with ForgeTree

---
In reply to: [505916607](https://github.com/microsoft/Forge/pull/32#discussion_r505916607) [](ancestors = 505916607)

Reply:
The "VisitNode" tests are at the top, followed by "WalkTree" tests.

Please remove this VisitNode test and add a new test just under TestTreeWalkerSession_WalkTree_Success.

---
In reply to: [505946715](https://github.com/microsoft/Forge/pull/32#discussion_r505946715) [](ancestors = 505946715,505916607)

Reply:
Just noticed the name ordering, I renamed the two cases and moved them into *WalkTree" session

---
In reply to: [506627465](https://github.com/microsoft/Forge/pull/32#discussion_r506627465) [](ancestors = 506627465,505946715,505916607)